drivers/gles3/shaders/scene.glsl

```diff
@@ -563,11 +563,17 @@ uniform highp samplerCubeShadow positional_shadow; // texunit:-4
 #endif // !defined(DISABLE_LIGHT_OMNI) || !defined(DISABLE_LIGHT_SPOT)
 
 #ifdef USE_MULTIVIEW
-uniform highp sampler2DArray depth_buffer; // texunit:-6
+uniform highp sampler2DArray depth_texture; // texunit:-6
```

Comment:
Will need a rebase, I renamed these to ``color_buffer`` and ``depth_buffer`` to match the other backends

Reply:
yeah I renamed to texture originally because we had that hardcoded in the compiler before all these changes.